.github/workflows/check-packages.yml

```diff
@@ -0,0 +1,21 @@
+  
+name: Build and run tests in container
+
+on:
+    schedule:
+        - cron: '0 0 * * 1'
+
+jobs:
+
+  build-run-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Build image
+      run: docker build . --file tests/Dockerfile --tag package-update-tests
```

Comment:
Think this needs to run from your `Dockerfile_no_lock`. Unless i am confusing myself.

Reply:
you are absolutely correct, not sure how I managed to omit that